kfs-core/src/main/resources/institutional-config.properties

```diff
@@ -22,6 +22,7 @@ property.files= \
   classpath:org/kuali/kfs/module/tem/tem-resources.properties, \
```

Comment:
You will need to incorporate changes from UAF-2096 removed white spaces from property.files. All property files names need to be on same line, with no spaces.